conans/model/conf.py

```diff
@@ -475,12 +479,20 @@ def __bool__(self):
 
     def get(self, conf_name, default=None, check_type=None):
         """
-        Get the value of the  conf name requested and convert it to the [type]-like passed.
+        Get the value of the conf name requested and convert it to the [type]-like passed.
         """
         pattern, name = self._split_pattern_name(conf_name)
         return self._pattern_confs.get(pattern, Conf()).get(name, default=default,
                                                             check_type=check_type)
 
+    def show(self, pattern):
+        result = {}
+
+        for p in self._pattern_confs.values():
```

Comment:
This is dropping the patterns if they exist, I think they should be shown, they are shown for options, for example, like ``*:shared=True`` is shown in the ``conan profile show`` command.